### ISSUE TITLE
Scale factor is reset to 1 after update

### ISSUE THREAD

**Issue by rhuiser** [bluemixgaragelondon/cf-blue-green-deploy#19]
When performing a blue-green deployment (update) on an app which has been scaled to multiple instances, the upgraded app is scaled to one while the old app remains scaled to its original number of instances.

Before blue-green-deploy:

~~~sh
name             requested state   instances   memory   disk   urls
static-app       started           5/5         64M      1G     static-app-macrodont-custos.cfapps.io
~~~

After blue-green-deploy:

~~~sh
name             requested state   instances   memory   disk   urls
static-app       started           1/1         64M      1G     static-app-macrodont-custos.cfapps.io
static-app-old   started           5/5         64M      1G
~~~

**Expected**: static-app instances 5/5, static-app-old instances 1/1

**Comment by timjacobi** [comment]
Thanks for reporting the issue. We will have a look into this.

**Comment by axelibm** [comment]
There is now a pull request (https://github.com/bluemixgaragelondon/cf-blue-green-deploy/pull/31) aiming to solve this issue

**Comment by timjacobi** [comment]
@rhuiser What are you using to scale the the application?

**Comment by rhuiser** [comment]
Currently just cf scale. We are I guess in the early stages exploring cloud foundry.

**Comment by timjacobi** [comment]
@rhuiser for now please see my comments here https://github.com/bluemixgaragelondon/cf-blue-green-deploy/pull/31#issuecomment-280188953

**Comment by bluemixgaragelondon** [comment]
Closed by #31. This will be available in cf repo binaries in the next release.
